cluster/gce/util.sh

```diff
@@ -741,7 +742,8 @@ function construct-common-kubelet-flags {
 # $1: if 'true', we're rendering flags for a master, else a node
 function construct-linux-kubelet-flags {
   local master="$1"
-  local flags="$(construct-common-kubelet-flags)"
+  local flags
+  flags="$(construct-common-kubelet-flags)"
```

Comment:
Quotes are superfluous in assignments like this. Try it out:

$ a=$(echo foo bar)           
$ echo $a
foo bar
$